crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -179,6 +180,9 @@ pub struct Node<S: TenTenOneStorage, N: Storage> {
     esplora_client: Arc<NodeEsploraClient>,
     pub pending_channel_opening_fee_rates: Arc<parking_lot::Mutex<HashMap<PublicKey, FeeRate>>>,
     pub probes: Probes,
+    // TODO: we should make this persistent as invoices might get paid later - but for now this is
+    // good enough
+    pub pending_usdp_invoices: Arc<parking_lot::Mutex<HashSet<bitcoin::hashes::sha256::Hash>>>,
```

Comment:
🤔 Do we really have to keep that list on the shared crate? I figure we could keep that on the app alone.

Reply:
You are right, can be stored some levels up . Ideally it was stored in the DB

Reply:
a9125f2a2812743a7adad436e0bfba09e824c5ff

Reply:
a9125f2a2812743a7adad436e0bfba09e824c5ff